deploy/docker/Dockerfile-alpine-python

```
@@ -0,0 +1,54 @@
+# Copyright 2021-2023 EMQ Technologies Co., Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG GO_VERSION=1.23.1
+FROM ghcr.io/lf-edge/ekuiper/base:$GO_VERSION-alpine AS builder
+
+COPY . /go/kuiper
+
+WORKDIR /go/kuiper
+
+RUN make build_with_edgex
+
+FROM python:3.12.7-alpine3.20
```

```suggestion
FROM python:3.12-alpine
```
I think it's better than `python:3.12.7-alpine3.20`, because we can use the latest patch version.